### PR TITLE
Prints error message on scene load failure

### PR DESCRIPTION
**Pull request by pjc0247**
Prints error messages properly

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

**Comment by vkalpias** [comment]
Hi,

Thanks for the PR. There was actually a very similar PR for the hierarchy resource handler a few days ago. Could you please have a look at src/resources/hierarchy.js and do the same for the error handling as that file? Or we can do it if you want. Thank you

**Comment by pjc0247** [comment]
I've take a look your changes,
It seems does not solve the problem.

Please check out my attachment, it's a very simple project which just throws something on `iniaitlize` method.
The error is still not visible.

image

[errtest.zip](https://github.com/playcanvas/engine/files/3180405/errtest.zip)

**Comment by pjc0247** [comment]
They have different behavior between editor and published project.

I can see the error message on the editor.
However, the stacktrace is totally mess. I can't recognize the error was thrown by `test.js`.

image

**Comment by vkalpias** [comment]
What I was saying was to change the scene handler to have the same error handler as the hierarchy handler.

**Comment by pjc0247** [comment]
Oh, got it. Sorry, I replied at 9AM, as soon as I sat in front of my PC. I was still sleepy,

I've committed the changes. Please take a look.

File: src/resources/scene-settings.js [hunk 1]
```diff
@@ -18,7 +18,17 @@ Object.assign(pc, function () {
                 if (!err) {
                     callback(null, response);
                 } else {
-                    callback("Error requesting scene: " + url.original);
+                    var errMsg = 'Error while loading scene ' + url.original;
```
Comment (vkalpias) [inline src/resources/scene-settings.js]: This should say Error while loading scene *settings*

**Pull request merged**